compiler/resolution/functionResolution.cpp

```diff
@@ -9877,9 +9888,9 @@ static void printUnusedFunctions() {
   std::map<FnSymbol*, std::vector<FnSymbol*> > instantiations;
 
   forv_Vec(FnSymbol, fn, gFnSymbols) {
-    if (FnSymbol* instantiatedFrom = fn->instantiatedFrom) {
-      while (instantiatedFrom->instantiatedFrom != NULL) {
-        instantiatedFrom = instantiatedFrom->instantiatedFrom;
+    if (FnSymbol* instantiatedFrom = toFnSymbol(fn->instantiatedFrom)) {
```

Comment:
Are these maybe changes from when you had function stub AST?

Reply:
Yeah, I'll remove them.